include/exadg/incompressible_navier_stokes/spatial_discretization/calculators/wall_shear_stress_calculator.cpp

```diff
@@ -0,0 +1,232 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#include <exadg/incompressible_navier_stokes/spatial_discretization/calculators/wall_shear_stress_calculator.h>
+
+namespace ExaDG
+{
+namespace IncNS
+{
+template<int dim, typename Number>
+WallShearStressCalculator<dim, Number>::WallShearStressCalculator()
+  : matrix_free(nullptr), dof_index(0), quad_index(0), dynamic_viscosity(0.0)
+{
+}
+
+template<int dim, typename Number>
+void
+WallShearStressCalculator<dim, Number>::initialize(
+  dealii::MatrixFree<dim, Number> const & matrix_free_in,
+  unsigned int const                      dof_index_in,
+  unsigned int const                      quad_index_in,
+  double const                            kinematic_viscosity,
+  double const                            density)
```

Comment:
why not writing the interface with `dynamic_viscosity`?

Reply:
I changed it, this was for no special reason. I will actually rewrite this once we have a fixed setup with the variable viscosity!